cmd/job-executor-service/main.go

```diff
@@ -160,6 +163,11 @@ func _main(args []string, env envConfig) int {
 		keptnOptions.UseLocalFileSystem = true
 	}
 
+	// Checking if the given job service account is empty
+	if env.DefaultJobServiceAccount == "" {
+		log.Println("WARNING: Using default service account for jobs!")
```

Comment:
We can try to make it clearer for the user since there are multiple defaults in play (at least they should have a hint at what they need to change to avoid it)
```suggestion
		log.Println("WARNING: No default service account for jobs configured: using kubernetes default service account!")
```